commands/create.go

```diff
@@ -215,10 +215,10 @@ func csvToMap(in []string) (map[string]string, error) {
 		}
 		for _, v := range fields {
 			p := strings.SplitN(v, "=", 2)
-			if len(p) != 2 {
+			if len(p) < 2 {
 				return nil, errors.Errorf("invalid value %q, expecting k=v", v)
 			}
-			m[p[0]] = p[1]
+			m[p[0]] = strings.Join(p[1:], "=")
```

Comment:
What is this change for?

Reply:
Environment variable values may contain `=` (and, frankly, `,`, which leads me to think the current approach will not do).

Reply:
It uses `SplitN` so it will never return more than 2 results.

Reply:
Fair point, I'll revert this change :+1: